libs/portal-integration-angular/assets/i18n/de.json

```diff
@@ -60,5 +60,8 @@
   },
   "OCX_HEADER": {
     "MENU_TOGGLE": "Menü"
-  }
+  },
+  "OCX_DIAGRAM": {
+      "SUMKEY": "Gesamtanzahl"
```

Comment:
OCX_DIAGRAM.SUM because sumKey is only the name of the variable to show that it is containing a translationkey